src/routes/(console)/project-[region]-[project]/sites/create-site/deploying/+page.svelte

```diff
@@ -18,9 +18,9 @@
     let deployment = $state(data.deployment);
 
     onMount(() => {
-        return sdk.forConsole.client.subscribe(
-            'console',
-            async (response: RealtimeResponseEvent<Models.Deployment>) => {
+        const unsubscribe = realtime
```

Comment:
not sure a project can subscribe to project `console`.
We have `sdk.forConsoleIn(region)` that can subscribe to `console` events on a correct region 👍 

also we can just return like ` return subscribe...`, no need for a variable 👍